src/Components/ClockComment.js

```diff
@@ -0,0 +1,14 @@
+import { Row } from 'react-bootstrap';
+
+/** function generating the text to be displayed above the clock
+ * @returns {Node} representing component
+ */
+function ClockComment() {
+  return (
+    <Row xs={1} className="justify-content-center m-auto">
+      <p className="fs-5 fst-italic text-center">More than halfway there!</p>
```

Comment:
### Nitpick

- Let's have this to have the string content to be more dynamic so let's make this get a text prop


```suggestion
      <p className="fs-5 fst-italic text-center">{props.text}</p>
```